configuration.nix

```diff
@@ -82,7 +82,8 @@ isMaximal: {
       dart.enable = false;
       ocaml.enable = false;
       elixir.enable = false;
-      haskell.enable = false;
+      haskell.enable = true;
+      hcl.enable = true;
```

Comment:
Please keep those disabled. `configuration.nix` dictates the servers enabled for the default packages, and Haskell toolchain is too large to be pulling for the default package.